README.Rmd

```diff
@@ -67,27 +67,30 @@ Because our office (mostly) cannot observe individual condo unit characteristics
 library(dplyr)
 library(tidyr)
 library(yaml)
-params <- read_yaml("params.yaml")
+library(httr)
+
+condo_params <- read_yaml("params.yaml")
+condo_preds <- condo_params$model$predictor$all
+
+res_params_text <- GET(
+  "https://raw.githubusercontent.com/ccao-data/model-res-avm/master/params.yaml"
+) %>%
+  content(as = "text")
+res_params <- read_yaml(text = res_params_text)
+res_preds <- res_params$model$predictor$all
+
+condo_unique_preds <- setdiff(condo_preds, res_preds)
+
 ccao::vars_dict %>%
-  filter(
-    var_is_predictor,
-    !var_name_model %in% c("meta_sale_price")
-  ) %>%
   inner_join(
-    as_tibble(params$model$predictor$all),
+    as_tibble(condo_preds),
     by = c("var_name_model" = "value")
   ) %>%
-  filter(
-    stringr::str_starts(var_name_model, "char_", negate = TRUE) |
-      var_name_model %in% c("char_yrblt", "char_land_sf") |
-      var_model_type == "condo",
-    stringr::str_starts(var_name_model, "ind_", negate = TRUE)
-  ) %>%
```

Comment:
I'm not entirely sure what this filter was trying to accomplish? But removing it doesn't seem to change the output, and it relied on the now-deprecated `var_model_type` column, so I stripped it out.

Reply:
I think originally this was for filtering all the variables in `vars_dict` down to only the things used by the condo model i.e. no characteristics except `yrblt` and `land_sf`. I don't _think_ this is necessary anymore given that `vars_dict` has been trimmed a bunch.